### ISSUE TITLE
le bot nutilise pas les bonnes couleurs

### ISSUE THREAD

**Issue by IIIYoucefYT** [IV2FI/DrawBot#39]
g deja essayer de changer les coordonne mais le bot utilise la premiere couleur que g mise et des couleurs random
que dois je faire?

**Comment by WizardRain62749-png** [comment]
yes please i have the same issue, i used a RGB color pallet to find the colors inside of the "paintcolors.txt" but even though the colors were all correct the system would put yellow where red and light blue are supposed to go and green for blue and blue for rose. what dose the system use to find the colors of the image, dose it have to be a 1 to 1 match in order for it to use the color?
